.eslintrc

```diff
@@ -1,6 +1,9 @@
  {
   "extends": "airbnb-base",
   "parser": "babel-eslint",
+  "env": {
```

Comment:
It should be possible to extend the .eslintrc file and add this environment for just the unit tests

Reply:
Yeah, let's do this in master/mono-repo branch even - not wait for the redesign to be merged.

Reply:
Created a ticket for it - [MLIBZ-2252](https://kinvey.atlassian.net/browse/MLIBZ-2252).